httpd/handlers/removePostTagVote.go

```diff
@@ -0,0 +1,119 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/jjcm/soci-backend/models"
+)
+
+// PostTagVoteRemoveRequest this is the shape of the JSON request that is needed to
+// create a vote for post tag
+type PostTagVoteRemoveRequest struct {
+	PostURL string `json:"post"`
+	TagName string `json:"tag"`
+}
+
+// RemovePostTagVote - protected http handler
+// the user associated with the passed auth token can create a new post-tag
+func RemovePostTagVote(w http.ResponseWriter, r *http.Request) {
+	// any non GET handlers need to attach CORS headers. I always forget about that
+	CorsAdjustments(&w)
```

Comment:
during some refactor/backlog sprint I thinkit'd be nice to clean up the various CORS code into a single place.

during some other dev work I discovered this package: [https://github.com/rs/cors](https://github.com/rs/cors)

could be a good fit to shorten up our handlers quite a bit.